v1beta1/proto/grafeas.proto

```diff
@@ -566,24 +566,18 @@ message GetVulnerabilityOccurrencesSummaryRequest {
 
 // A summary of how many vulnerability occurrences there are per severity type.
 message VulnerabilityOccurrencesSummary {
-  // A map of how many occurrences were found for each severity.
-  repeated SeverityCount counts = 1;
   // A listing by resource of the number of fixable and total vulnerabilities.
-  repeated FixableTotalByDigest fixable_counts = 2;
-  // The number of occurrences for a specific severity.
-  message SeverityCount {
-    // The severity of the occurrences.
-    grafeas.v1beta1.vulnerability.Severity severity = 1;
-    // The number of occurrences with the severity.
-    int64 count = 2;
-  }
-  // Per resource counts of fixable and total vulnerabilites.
+  repeated FixableTotalByDigest fixable_counts = 1;
```

Comment:
just count now? since we've combined them

Reply:
or "counts" rather

Reply:
done